engine/platform/src/platform_window_glfw.cpp

```diff
@@ -362,20 +362,45 @@ namespace dmPlatform
         return -1;
     }
 
+    #ifndef __EMSCRIPTEN__
+        #define GLFW_AUX_CONTEXT_SUPPORTED
+    #endif
+    static inline int32_t QueryAuxContextImpl()
+    {
+    #if defined(GLFW_AUX_CONTEXT_SUPPORTED)
+        return glfwQueryAuxContext();
```

Comment:
I guess we could add this function to the GLFW js library, but I don't want to keep adding things to it since it's an external lib nowadays.